view/layout/index.css

```diff
@@ -12,6 +12,7 @@
       "C"
       "H"
       "A"
+      "plot3d"
```

Comment:
Let’s rename to `P` to have one-letter codes

Reply:
there is already P grid-area

Reply:
`3`?

Reply:
P3 or just 3 ?
Imagine being a new guy, that reading this code. You will need some research to realise what does it mean

Reply:
Yes, we need a refactoring here, but let’s it do consistently, renaming all nodes.

Reply:
I created an issue https://github.com/evilmartians/oklch-picker/issues/70

Just `3` for now (if CSS allows starting name from the digit)